README.md

```diff
@@ -17,169 +17,9 @@ Reading the short [Basic Terms](https://github.com/FeatureProbe/FeatureProbe/blo
-}
-```
-
-### Step 3. Use the feature toggle
-
-You can use sdk to check which variation a particular user will receive for a given feature flag.
-
-```java
-public class Demo {
-    private static final FPConfig config = FPConfig.builder()
-            // FeatureProbe server URL for local docker
-            // .remoteUri("http://127.0.0.1:4007")
-            // FeatureProbe server URL for featureprobe.io
-            .eventUrl(new URL("https://featureprobe.io/server/api/events"))
-            .synchronizerUrl(new URL("https://featureprobe.io/server/api/server-sdk/toggles"))
-            .pollingMode(Duration.ofSeconds(3))
-            .useMemoryRepository()
-            .build();
-
-    private static final FeatureProbe fpClient = new FeatureProbe("server-8ed48815ef044428826787e9a238b9c6a479f98c",
-            config);
-
-    public void test() {
-        String uniqueUserId = /* uniqueUserId */;
-        FPUser user = new FPUser(uniqueUserId).with("city", /* city */).with("gender", /* gender */).with("userId", /* userId */);
-        boolean boolValue = fpClient.boolValue("bool_toggle_key", user, false);
-        if (boolValue) {
-            // application code to show the feature
-        } else {
-            // the code to run if the feature is off
-        }
-    }
-}
-```
-
-## Testing
-
-We have unified integration tests for all our SDKs. Integration test cases are added as submodules for each SDK repo. So
-be sure to pull submodules first to get the latest integration tests before running tests.
-
-```shell
-git pull --recurse-submodules
-mvn test
-```
-
-## Mock
-
-### 1、Add powermock SDK to your project:
-
-```xml
-<dependency>
-    <groupId>org.powermock</groupId>
-    <artifactId>powermock-api-mockito2</artifactId>
-    <version>2.0.9</version>
-    <scope>test</scope>
-</dependency>
-<dependency>
-    <groupId>org.powermock</groupId>
-    <artifactId>powermock-module-junit4</artifactId>
-    <version>2.0.9</version>
-    <scope>test</scope>
-</dependency>
-```
-
-### 2、Mock Toggle 
-
-#### *target method*
-```java
-@AllArgsConstructor
-@Service
-public class DemoService {
-
-    FeatureProbe fp;
-
-    public boolean isTester(String userId, String tel) {
-        FPUser fpUser = new FPUser(userId);
-        fpUser.with("tel", tel);
-        return fp.boolValue("is_tester", fpUser, false);
-    }
-}
-```
-#### *unit test*
-```java
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({FeatureProbe.class})
-public class FeatureProbeTest {
-
-    @Test
-    public void test() {
-        FeatureProbe fp = PowerMockito.mock(FeatureProbe.class);
-        DemoService demoService = new DemoService(fp);
-        Mockito.when(fp.boolValue(anyString(), any(FPUser.class), anyBoolean())).thenReturn(true);
-        boolean tester = demoService.isTester("user123", "12397347232");
-        assert tester;
-    }
-
-}
-```
+## How to Use This SDK
+
+See [SDK Doc](https://docs.featureprobe.io/sdk/Server-Side%20SDKs/java-sdk) for detail.
```

Comment:
@HeZean   readme detail has been move to https://github.com/FeatureProbe/feature-probe-docs/blob/main/docs/sdk/Server-Side%20SDKs/java-sdk.md , if you have time please help to also update the ReadmeUpdater script and action.